internal/rbd/rbd_journal.go

```diff
@@ -25,6 +25,13 @@ import (
 	"github.com/ceph/ceph-csi/internal/util"
 )
 
+const (
+	// PVVolumeHandleLabel is the annotation key set on the PV object.
+	PVVolumeHandleLabel = "CephCSIVolumeHandle"
```

Comment:
Labels are commonly formatted like `csi.ceph.io/volume-handle`. All lowercase, with a prefix (`csi.ceph.io/`) for the application/component, and sub-label (`volume-handle`) as real 'label'.

It would be good to adapt such a format here too.

Reply:
This is for annotation sorry need to change the name. is it suggested to use `csi.ceph.io/volume-handle` for annotation too?

Reply:
Yes, I think so. Identifying the application/component that set a Label or Annotation seems to be good practice.